### DISCUSSION TITLE
Individual Email Alert

### DISCUSSION THREAD

**Discussion by schnudd31do3** [snipe/snipe-it#12073] [Q&A]
We have assets in use that are out of warranty. We want to continue managing these assets in Snipe-IT. Unfortunately, there is no way to individually acknowledge the alarm messages for assets or to switch them off individually. This results in unnecessary alarm messages. Maybe there is a setting that will help me and I haven't found it yet.

**Comment by schnudd31do3** [comment]
It is a pity that no one has been interested in this topic until now. Too bad I didn't even get a single reply. So I'm closing this now.

**Comment by marcusmoore** [comment]
Hi @schnudd31do3, this is something we're interested in and are internally planning an overhaul on how notifications work. There's a good possibility that functionality would be included in the changes.

Unfortunately it's not something that is going to ship tomorrow but it is tentatively planned for v7.
